processes/process_sleepspindle_parameters/process_sleepspindle_parameters

```diff
@@ -0,0 +1,202 @@
+    TimeNP = zeros (Nchannels,Nepochs); % Time of max negative peak
+
+    % ===== LOAD THE DATA =====
+    for iNepochs = 1 : Nepochs
+       DataMat   = in_bst(sInputs(iNepochs).FileName, [], 0);
+       epochSize = size(DataMat.F);  
+       Ntime     = epochSize(2); % Number of time samples changes depending on sleep spindle
+       Data      = DataMat.F; % Actual data containing EEG, EOG and EMG data
+       Time      = DataMat.Time(end); % Time information of the recording
+       Fs        = (Ntime-1)/Time; % Calculation of Sampling frequency
+
+    % ===== PROCESS =====
+    % This is where the actual process of data manipulation and calculation takes place.
+       DataN        = Data*-1; % The negative version of the data
+       Data(end,:)  = (randi(100,1,Ntime))*10^-6; % End row which is BDF/status channel is a constant number so replaced this with randi to remove issues during findpeaks 
+       DataN(end,:) = (randi(100,1,Ntime))*10^-6;
+       Data_Chan    = zeros(1,Ntime); % Row vector with Ntime X zeros
+       DataN_Chan   = zeros(1,Ntime); % Row vector with Ntime X zeros
+
+            for i = 1 : Nchannels
+                Data_Chan(1,:)     = Data(i,:); % Extracts individual channel info from the data matrix
+                DataN_Chan(1,:)    = DataN(i,:); % Extracts individual channel info from the negative data matrix
+                [PosP, LocP]       = findpeaks(Data_Chan,Fs); % Find all the positive peaks and their locations in seconds
+                [NegP, LocN]       = findpeaks(DataN_Chan,Fs); % Find all the negative peaks and their locations in seconds
+                [PeakPo(i,iNepochs), IndexP] = max(PosP,[],2); % Find the max positive peak per channel and index its location 
+                [PeakNe(i,iNepochs), IndexN] = max(NegP,[],2); % Find the max negative peak per channel and index its location
+                TimePP(i,iNepochs) = LocP(IndexP); % Uses the IndexP vector to find the time point of max positive peak
+                TimeNP(i,iNepochs) = LocN(IndexN); % Uses the IndexN vector to find the time point of max negative peak (won't be used further from here though)
+                CountP(i,iNepochs) = length(PosP); % Count the number of peak 
+            end
+
+       SS_Dur(:,iNepochs) = Time; % Calculation of spindle duration
+       SS_Fre             = 1./(SS_Dur./CountP); % Final calculation of spindle frequency
+       SS_Amp             = PeakPo + PeakNe; % Final calculation of spindle max peak-peak amplitude
+       SS_Sym             = (TimePP./SS_Dur)*100; % Final calculation of spindle symmetry
+
+    % Spindle duration = the number of time samples-1 divided by the sampling frequency of 1024 
+    % Spinde frequency = the reciprocal of (the number of peaks/duration)
+    % Spinde amplitude = the sum of max positive peak and negative peak
+    % Spindle symmetry = the percentage of (the time location of max positive peak/spindle duration) 
```

Comment:
Each of these comments would be better placed in the line before the feature computation

Reply:
updated as appropriate